index.html

```diff
@@ -663,18 +671,18 @@ <h3>Prohibited States and Properties</h3>
+			<p>Descendents which are not direct children of an element ancestor are not constrained by <em>allowed child roles</em>. For example, an <code>image</code> is not an allowed child of a <code>list</code>, but it is a valid descendent so long as it is also a descendant of the <code>list</code>'s allowed child <code>listitem</code>.</p>
+			<p class="note">A role that has 'allowed child roles' does not imply the reverse relationship. Elements with roles in this list do not always have to be found within elements of the given role. See <a href="#scope">allowed parent roles</a> for requirements about the context where elements of a given role will be contained.</p>
+			<p class="note">An element with a <a href="#subclassroles">subclass role</a> of the 'allowed child role' does not fulfill this requirement. For example, the <rref>listbox</rref> role allows a child element using the <rref>option</rref> or <rref>group</rref> role. Although the <rref>group</rref> role is the superclass of <rref>row</rref>, adding a child element with a role of <rref>row</rref> will not fulfill the requirement that <rref>listbox</rref> allows children with <rref>option</rref> or <rref>group</rref> roles.</p>
 			<p class="note">An element with the appropriate <a href="#implicit_semantics">implicit WAI-ARIA semantic</a> fulfills this requirement.</p>
 		</section>
 		<section id="scope">
-			<h3>Required Context Role</h3>
-			<p>The required context role defines the owning container where this <a>role</a> is allowed. If a role has a required context, authors MUST ensure that an element with the role is contained inside (or <a>owned</a> by) an element with the required context role. For example, an element with role <code>listitem</code> is only meaningful when contained inside (or <a>owned</a> by) an element with role <code>list</code>.</p>
-			<p class="note">A role that has 'required context role' does not imply the reverse relationship. While an element with the given role needs to appear within an element of the listed role(s) in order to be meaningful, elements of the listed roles do not always need descendant elements of the given role in order to be meaningful. See <a href="#mustContain">required owned elements</a> for requirements about elements that require presence of a given descendant to be processed properly.</p>
+			<h3>Required Parent Role</h3>
+			<p>The required <a>parent</a> role defines the parent container where this <a>role</a> is allowed. If a role has a required parent, authors MUST ensure that an element with the role is a direct <a>child</a> of an element with the required parent role. For example, an element with role <code>listitem</code> is only meaningful when it is a child of an element with role <code>list</code>.</p>
+			<p>To determine whether an element has a parent with the required role, <a>user agents</a> MUST ignore any elements with the role <rref>generic</rref> or <rref>presentation</rref>.</p>
```

Comment:
I thought we discussed this at some point and decided not to be prescriptive about it? I don't remember exactly, though.

Reply:
not sure?  i don't recall that discussion.   but to quickly illustrate my point:

```
<div role=list>
  <div tabindex=-1>
    <div role=listitem>1</div>
    <div role=listitem>2</div></div>
</div>
```

because there is a `tabindex` on the `div`, chromium is presently calculating a name for the `div` from its children.  With VoiceOver, this list is now exposed as a "list of 1 item" and its direct child is a "group" with the name "1 2".